src/index.js

```diff
@@ -0,0 +1,163 @@
+import './styles/styles.css';
+import TaskObject from '../modules/taskObj.js';
+// eslint-disable-next-line import/no-cycle
+import clearCompletedTask from '../modules/clearCompletedTask.js';
+import sendToLocalAgain from '../modules/updateLocStore.js';
+
+const refresh = document.querySelector('.uil-sync');
+const input = document.querySelector('input');
+const todosParent = document.querySelector('.todos-container');
+const clearCompletedTaskBtn = document.querySelector('button');
+
+// Function to Delete Tasks
+const deleteTask = (todo) => {
+  // Delete task from browser
+  todosParent.removeChild(todo);
+
+  // Delete from localStorage
+  const localStoreData = JSON.parse(localStorage.getItem('list'));
+  const data = Array.from(localStoreData).filter((task) => task.complete === false);
+  data.map((todo, index) => {
+    todo.index = index + 1;
+    return todo;
+  });
+  localStorage.setItem('list', JSON.stringify(data));
+};
+
+const myArray = [];
```

Comment:
- [ ] As the previous reviewer mentioned, kindly add some examples (simples) to the `myArray` array
e.g.
```
const myArray = [
  {
    description: 'An example description in string',
    completed: false,
    index: 1,
  },{
    description: 'An example description in string',
    completed: false,
    index: 2,
  },


```